src/test/resources/data/FeedbackSessionQuestionTypeTest.json

```diff
@@ -631,7 +631,7 @@
       "courseId": "FSQTT.idOfTypicalCourse1",
       "creatorEmail": "instructor1@course1.tmt",
       "questionMetaData": {
-        "value": "{\"numOfMcqChoices\":2,\"mcqChoices\":[\"It's good\",\"It's perfect\"],\"questionText\":\"What do you like best about the class' product?\",\"questionType\":\"MCQ\",\"otherEnabled\":false}"
+        "value": "{\"hasAssignedWeights\": false,\"mcqWeights\": [],\"mcqOtherWeight\": 0.0,\"numOfMcqChoices\": 2,\"mcqChoices\": [\"It's good\",\"It's perfect\"],\"otherEnabled\": false,\"generateOptionsFor\": \"NONE\",\"questionType\": \"MCQ\",\"questionText\": \"What do you like best about the class' product?\"}"
```

Comment:
Have you deliberately left some metadata without `mcqWeights` etc?

Reply:
Yes, instead of modifying the existing questions to add weights, I created a new question for that purpose, in this way, all the cases which used the previous questions will not get affected,